src/Databases/PostgreSQL/DatabasePostgreSQL.cpp

```diff
@@ -39,14 +39,16 @@ DatabasePostgreSQL::DatabasePostgreSQL(
         const String & metadata_path_,
         const ASTStorage * database_engine_define_,
         const String & dbname_,
-        const String & postgres_dbname,
+        const String & postgres_dbname_,
```

Comment:
If class is called Postgres it does not make sense to clarify members with postgres.

Reply:
it makes sence because of dbname as there can be two different database names: database name in clickhouse and database name in postgresql.

Reply:
What about postgres_schema ?

Reply:
Also I am not sure that `postgres_dbname` should be a member because IDatabase is initialised with  `postgres_dbname` and  `database_name`later can be accessed as protected member.

Reply:
> because IDatabase is initialised with postgres_dbname

No, it is initialized with `dbname` - clickhouse database name, not postgres_dbname, they are different.

>  and database_name later can be accessed as protected member.

Yes, but there is no member in DatabasePostgreSQL which is database_name. 

> What about postgres_schema ?

For uniformity with postgres_dbname.

Reply:
Yes, make sense.